utils.go

```diff
@@ -0,0 +1,24 @@
+package clustertest
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func GenerateRandomName(prefix string) string {
```

Comment:
have you considered pkg `k8s.io/kubernetes/test/e2e/framework`?
It also contains a `RandomSuffix` func we could use instead this one.

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L521-L524

Reply:
I haven't yet as there wasn't enough value pulling in the whole package. 

There's also discussions upstream about pulling that framework out into it's own stand-alone repo for use like this but there's still ongoing work before that is possible.